tools/grab_starred_links.py

```
@@ -63,22 +66,21 @@
-            print(f"Skipping invalid entry: {title}, {url}")
+            logging.warning(f"Skipping invalid entry: {title}, {url}")
             return False
         
         slug = generate_unique_slug(title, date, url)
         file_path = os.path.join(HUGO_CONTENT_DIR, f"{slug}.md")
         
         if os.path.exists(file_path):
-            print(f"Skipping existing entry: {title}")
+            logging.info(f"Skipping existing entry: {title}")
             return False
         else:
```

Ruff (0.8.2)

Unnecessary `else` after `return` statement

Remove unnecessary `else`

(RET505)